misk-metrics/src/main/kotlin/misk/metrics/Histogram.kt

```diff
@@ -14,18 +15,32 @@ import java.util.concurrent.TimeUnit
  * A sample implementation can be found in PrometheusHistogram
  */
 interface Histogram {
+
+  fun getHistogram(): Summary
```

Comment:
there's a few internal implementations of Histogram who this will be backwards incompatible for